dive_core/gfxr_vulkan_command_hierarchy.cpp

```diff
@@ -133,12 +148,17 @@ const DiveAnnotationProcessor::VulkanCommandInfo &vk_cmd_info)
 
 //--------------------------------------------------------------------------------------------------
 bool GfxrVulkanCommandHierarchyCreator::ProcessVkCmds(
-const std::vector<DiveAnnotationProcessor::VulkanCommandInfo> &vkCmds)
+const std::vector<DiveAnnotationProcessor::VulkanCommandInfo> &vkCmds,
+const std::vector<uint64_t>                                   &draw_call_counts,
+const std::vector<uint64_t>                                   &render_pass_draw_call_counts)
 {
+    std::vector<uint64_t> mutable_draw_call_counts = draw_call_counts;
```

Comment:
why not just pass in std::vector<uint64_t>& as arg directly?

Reply:
I created the mutable counts since the const vectors belong to the const GfxrCaptureData. I could make GfxrCaptureData passed to ProcessGfxrSubmits() non-const which would allow me to pass in std::vector<uint64_t>& as arg directly